doc/admin/conf_files/kdc_conf.rst

```diff
@@ -60,6 +60,10 @@ subsection does not contain a relation for the tag.  See the
     Specifies the maximum packet size that can be sent over UDP.  The
     default value is 4096 bytes.
 
+**kdc_tcp_listen_backlog**
+    (Integer.)  Set the size of the listen() queue length for the KDC
+    daemon. The value may be limited by OS settings. The default value
+    is 5.
```

Comment:
Above, "With one exception" needs to be changed to "with two exceptions."  Also, two spaces after periods.